core/scripts/vrfv2/testnet/main.go

```diff
@@ -161,117 +157,117 @@ func main() {
 			uint16(5),       // confs
 			uint32(1),       // words
 			keyHashBytes)
-		panicErr(err)
+		helpers.PanicErr(err)
 		fmt.Println("Consumer address", consumerAddress, "hash", tx.Hash())
 	case "consumer-subscribe":
 		consumerSubscribeCmd := flag.NewFlagSet("consumer-subscribe", flag.ExitOnError)
 		consumerSubscribeAddress := consumerSubscribeCmd.String("address", "", "consumer address")
-		panicErr(consumerSubscribeCmd.Parse(os.Args[2:]))
+		helpers.ParseArgs(consumerSubscribeCmd, os.Args[2:], "address")
 		consumer, err := vrf_single_consumer_example.NewVRFSingleConsumerExample(common.HexToAddress(*consumerSubscribeAddress), ec)
-		panicErr(err)
+		helpers.PanicErr(err)
 		tx, err := consumer.Subscribe(owner)
-		panicErr(err)
+		helpers.PanicErr(err)
 		fmt.Println("hash", tx.Hash())
 	case "link-balance":
 		linkBalanceCmd := flag.NewFlagSet("link-balance", flag.ExitOnError)
 		linkAddress := linkBalanceCmd.String("link-address", "", "link-address")
 		address := linkBalanceCmd.String("address", "", "address")
-		panicErr(linkBalanceCmd.Parse(os.Args[2:]))
+		helpers.ParseArgs(linkBalanceCmd, os.Args[2:], "link-address", "address")
 		lt, err := link_token_interface.NewLinkToken(common.HexToAddress(*linkAddress), ec)
-		panicErr(err)
+		helpers.PanicErr(err)
 		b, err := lt.BalanceOf(nil, common.HexToAddress(*address))
-		panicErr(err)
+		helpers.PanicErr(err)
 		fmt.Println(b)
 	case "consumer-cancel":
 		consumerCancelCmd := flag.NewFlagSet("consumer-cancel", flag.ExitOnError)
 		consumerCancelAddress := consumerCancelCmd.String("address", "", "consumer address")
-		panicErr(consumerCancelCmd.Parse(os.Args[2:]))
+		helpers.ParseArgs(consumerCancelCmd, os.Args[2:], "address")
 		consumer, err := vrf_single_consumer_example.NewVRFSingleConsumerExample(common.HexToAddress(*consumerCancelAddress), ec)
-		panicErr(err)
+		helpers.PanicErr(err)
 		tx, err := consumer.Unsubscribe(owner, owner.From)
-		panicErr(err)
+		helpers.PanicErr(err)
 		fmt.Println("hash", tx.Hash())
 	case "consumer-topup":
 		// NOTE NEED TO FUND CONSUMER WITH LINK FIRST
 		consumerTopupCmd := flag.NewFlagSet("consumer-topup", flag.ExitOnError)
 		consumerTopupAmount := consumerTopupCmd.String("amount", "", "amount")
```

Comment:
Lets make the description of this flag "amount in Juels" instead of just "amount" to make the units explicit.